src/app/components/algorithms/impl-selection-criteria/impl-selection-criteria.component.ts

```diff
@@ -222,22 +275,31 @@ export class ImplSelectionCriteriaComponent implements OnInit, OnChanges {
           )
       );
     } else {
-      body = {
-        name: this.impl.name,
-        implementedAlgorithm: this.algo.id,
-        selectionRule: '',
-        // TODO
-        sdk: 'Qiskit',
-        language: 'OpenQASM',
-        fileLocation: 'http://example.com/',
-      };
-      this.nisqImplementationService
-        .createImplementation({ body })
-        .subscribe((newImpl) => {
-          this.nisqImpl = newImpl;
-          this.oldNisqImpl = cloneDeep(newImpl);
-          this.selection.clear();
-        });
+      this.sdkService.getSdks().subscribe((dto) => {
+        if (dto.sdkDtos.filter((sdk) => sdk.name === 'Qiskit').length < 1) {
```

Comment:
if Qiskit is not in nisq analyzer, also check if it is in qc atlas and add it also there if not available